src/Orchard.Cms.Web/Modules/Orchard.Settings/Views/Admin/RestartSite.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿<div class="alert alert-warning">
```

Comment:
is this file in correct place?  

Wouldn't '/src/OrchardCore.Modules/OrchardCore.Settings/Views/Admin/RestartSite.cshtml' be a better place?

Reply:
This PR is outdated. I submitted it on April but it was not merged, so many changes in Orchard since then. Currently I'm a busy on other things to invest time on update it to latest bits of Orchard. Feel free to put a new PR for latest bits